docs/api_docs/template/members.tpl

```diff
@@ -0,0 +1,48 @@
+{{ define "members" }}
```

Comment:
Add comment about from where we are referring these files.
Also add license header of original repo if required.

Reply:
@PrasadG193 , 

- Do I need to add comments on all of the files which we imported as to where they are being used?

- Where and how do I add the license?(I haven't done that before, so any insight would be helpful)

Reply:
@Kartik-Garg lets add to the files you have copied as-it-is. Just a comment should be enough

Reply:
@PrasadG193 Added comments on the imported files, except the `config-json` file. Since, JSON files do not support comments.